cruise.umple/src/Umple_Code.ump

```diff
@@ -18,6 +18,15 @@ namespace cruise.umple.compiler;
 class UmpleModel
 {
 
+    //ParsingTime represents Parsing end time
+    //gTime1 represents code generation start time
+    //gTime2 represents code generation end time
+    Double parsingTime = 0.0;
+    Double gTime1 = 0.0;
```

Comment:
`gTime1` and `gTime2` must be indicated as `internal`. The reason is that there is no need to them to be available through an instance of `UmpleModel`